ChatSX/screens/ChatRoom/index.tsx

```diff
@@ -122,6 +125,10 @@ import {FlashList} from '@shopify/flash-list';
 import WarningMessageModal from '../../customModals/WarningMessage';
 import {SyncConversationRequest} from 'reactnative-chat-data';
 import {useQuery} from '@realm/react';
+import LinearGradient from 'react-native-linear-gradient';
+import {createShimmerPlaceholder} from 'react-native-shimmer-placeholder';
+import {paginatedSyncAPI} from '../../utils/syncChatroomApi';
+const ShimmerPlaceHolder = createShimmerPlaceholder(LinearGradient);
```

Comment:
is this import statement?

Reply:
no, here creating shimmerPlaceHolder ref. will add a line space for diff